src/bin/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@ target_link_libraries(msquic PRIVATE core platform)
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     target_link_libraries(msquic PUBLIC
         ws2_32 schannel ntdll bcrypt ncrypt crypt32 iphlpapi advapi32)
+    if(QUIC_UWP_BUILD)
+        target_link_libraries(msquic PUBLIC OneCoreUAP)
```

Comment:
Just go ahead and add this to the list above unconditionally, assuming it works.

Reply:
This would break compatibility with Windows 7, since that is a UAP specific library.

Reply:
I'm fine that that, as `SetThreadDescription` is also newer than Win7. We have no goals to support < Win10.

Reply:
Ok. So no goals to support Windows 8.1 either? If so, we should change everything to link to WindowsApp.lib, which has all of those APIs.

Reply:
Ok. Turns out I still need this separate. I can use WindowsApp.lib instead of many of the other windows APIs, but GetActiveProcessor account only exists in the library for UWP, and linking to it from a desktop app will not work correctly.